documentation/domains/Domain.json

```diff
@@ -374,7 +374,7 @@
           "$ref": "#/definitions/ServerService"
         },
         "domainHome": {
-          "description": "The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is FromModel.",
+          "description": "The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is FromModel.",
```

Comment:
The sentence "Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image" needs to be modified to append the domainUID directory name.

Reply:
What's listed is the correct value when the domainHomsSourceType is image. The reason is that we have customers who want to use the same image for different domains, which would have different domainUIDs. Therefore, we accept the domainHome in this case either being at or just above the actual domainHome value.

Reply:
I've updated the text. Let me know what you think.